gpio.go

```diff
@@ -0,0 +1,19 @@
+package drivers
+
+// OutputPinner represents a pin used for digital output.
+type OutputPinner interface {
+	//Configure sets the pin as output.
+	Configure() error
+
+	// Set sets the output to the given state.
+	Set(state bool) error
+}
+
+// InputPinner represents a pin used for digital input.
+type InputPinner interface {
+	// Configure sets the pin as input. Additionally mode can be set, such as an internal pull up or pull down resistor.
+	Configure() error
```

Comment:
we could alternatively define ConfigurePullUp(), ConfigurePullDown() function

Reply:
This is already discussed in #749. A recent proposal is given in #753 . So we will follow the consent of that PR.